util/libcrypto.num

```diff
@@ -4393,3 +4392,18 @@ CRYPTO_THREAD_glock_new                 4336	1_1_1	EXIST::FUNCTION:
 UI_get_result_length                    4337	1_1_1	EXIST::FUNCTION:
 UI_set_result_ex                        4338	1_1_1	EXIST::FUNCTION:
 UI_get_result_string_length             4339	1_1_1	EXIST::FUNCTION:
+RAND_POOL_new                           4345	1_1_1	EXIST::FUNCTION:
```

Comment:
Look like a manual editing, as this numbers are normally consecutive.

Reply:
I was fed up with constantly fixing collisions with master ;-). But I will renumber them to make them consecutive.